### PR TITLE
LPS-68390 gradlew buildLang

### PR DESCRIPTION
**Pull request by brandizzi**
Hey Bruno and @thiago-rocha!

I guess displaying the date in the user's time zone is sensible, since we do it elsewhere in the portal (although I feel it is not my decision :) ).

If we are going to do it. to use the `toString()` method from JS `Date` would fail. It uses the browser's time zone. So we would to make the conversion on the server-side (or add a whole library to such conversions!) because JS do not support arbitrary time zone conversion.

Not only that, `toString()` will display an locale-arbitrary representation of the date, generally in English. So it is untranslatable. There are other methods , such as `Date.toLocaleString()`, but it would use the language of the browser, so we could end up with things such as "Rascunho salvo em Oct 13, 2016 etc. etc." For this reason, I believe it is better to delegate the translation to a Language.properties mask.

Summing up, this is my proposal for a translatable, localizable message. The original PR is perfectly function, but untranslatable. So, if you find this fix too complicated, we can postpone it to later, when we are going to translate the messages.

Let me know of your opinions! Thanks!

### CODE REVIEW

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
Pull request submitted to https://github.com/vicnate5/liferay-portal/pull/598. [See changes here.](https://github.com/brunobasto/liferay-portal/compare/931ea1d~1...931ea1d) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brunobasto** [comment]
After talking to @brandizzi, we decided to tackle the message format later.
